contrib/images/ibcsim-bcd/Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV PATH $GOPATH/bin:$PATH
 
 ENV GO111MODULE on
 ENV RELAYER_TAG v2.5.2
-ENV BABYLON_SDK_TAG v0.6.0-rc.0
+ENV BABYLON_SDK_VERSION main
```

Comment:
What about releasing `v0.6.0-rc.1`. I know it's not semantically version correct, but better than this, which defeats versioning altogether.

Reply:
Good idea, let's do this